cds/modules/records/templates/cds_records/record_stats.html

```diff
@@ -31,10 +31,16 @@
 <div class='container'>
   <div class='row'>
     <div class='col-sm-12'>
-      <h2>Pageviews</h2>
-      <div class='pageviews'></div>
+      <h2>Page Views</h2>
+      <div class='pageviews'>
+        <i id="pageviews-loading-spinner" class="fa fa-spinner fa-spin"></i>
+        <p id="pageviews-error-message" style="display: none;">Something went wrong while fetching the data... Please try again.</p>
```

Comment:
Minor: IMHO, I would not show that we are so bad at it :)
I will maybe put something like:
"There was a problem while rendering this data and we are investigating the issue"
Maybe Esteban has better ideas! :)